### PR TITLE
Included Kifi recevied time in search events + better naming

### PR DESCRIPTION
**Pull request by stkem**
@leogrim @2is10

### CODE REVIEW

File: packrat/scripts/google_inject.js [hunk 1]
```diff
@@ -284,8 +285,9 @@ if (searchUrlRe.test(document.URL)) !function() {
           "experimentId": response.experimentId,
           "kifiResults": response.hits.length,
           "kifiCollapsed": !response.expanded,
-          "kifiTime": tKifiResultsShown - tQuery,
-          "referenceTime": tGoogleResultsShown - tQuery,
+          "kifiTime": tKifiResultsReceived - tQuery,
+          "kifiShownTime": tKifiResultsShown - tQuery,
+          "googleShownTime": tGoogleResultsShown - tQuery,
```
Comment (2is10) [inline packrat/scripts/google_inject.js]: It seems like up until now we’ve been keeping the event tracking JSON key names independent of the search provider. Does it matter?

Comment (leogrim) [inline packrat/scripts/google_inject.js]: You're right, bad suggestion on me. 

referenceShownTime? 
thirdPartyShownTime?
(I have been using "thirdParty" in Mixpanel)

Comment (stkem) [inline packrat/scripts/google_inject.js]: @leogrim Your call, I think both of these are fine.

Comment (leogrim) [inline packrat/scripts/google_inject.js]: Let's go for thirdPartyShownTime

Comment (stkem) [inline packrat/scripts/google_inject.js]: Done

**Pull request merged**